daemon/indexing/listener/listener.js

```diff
@@ -136,6 +139,26 @@ async function liveTracking(config) {
   check()
 }
 
+function getLastBlock(config){
+  if(config.continueFile == undefined || !fs.existsSync(config.continueFile)){
+    return 0
```

Comment:
If config file not found, I would print an error and exit rather than defaulting to block 0

Reply:
It's going to be very normal that there's no continue file the first time you run it. Also for now if you want to reindex, you can just delete the continue file.